index.org

```diff
@@ -90,6 +114,40 @@ sur la licence d’un logiciel avec la technique précendente, il faut se
 rendre sur le dépôt de code source du logiciel. La licence doit se
 trouver soit à la racine du dépôt soit dans un répertoire =LICENSES=.
 
+*** Analyser : les autres aspects 
+
+Si la licence est l'élément indispensable que l'on vérifie 
+avant d'aller plus loin, il en est trois autres à ne pas négliger : les fonctionnalités, la sécurité et la communauté. 
+
+**** Fonctionnalités 
+
+Il n'est pas rare qu'un logiciel ne couvre pas entièrement les besoins fonctionnels visés. 
+Et c'est là que la magie opère : vous pouvez [[#contribuer-a-un-logiciel-libre][contribuer]] 
+au projet et ainsi participer à l'effort collectif, récompenser la communauté. 
+
+Il est donc utile de distinguer au sein des fonctionnalités attendues, 
+celles qui sont couvertes et celles qui ne le sont pas. Parmi ces 
+dernières, il est utile de mettre en balance leur criticité pour vous 
+et leur moyen d'intégration dans le projet (ex. "Est-ce facile à intégrer 
+via des plugins ou cela nécessite-t-il une modification du coeur ?"). 
+
+**** Sécurité 
+
+La sécurité est un sujet que l'on ne peut plus ignorer. Cet aspect d'un projet 
+peut être analysé sur la base de divers critères. 
+La OpenSSF dresse la liste des [[https://www.bestpractices.dev/fr/criteria][critères]] 
+qu'elle utilise pour labelliser des projets. C'est une base de départ solide. 
+
+**** Communauté 
```

Comment:
On ne veut pas donner l'impression que (1) tous les projets libres ont ou doivent avoir des communautés et (2) qu'un projet avec une communauté est "meilleur" qu'un projet sans.

Mais on peut dire qu'un projet avec une communauté (d'utilisateurs ? de contributeurs ?) active donne prise à des diagnostics distincts : vérifier la façon dont les contributeurs répondent à la communauté (etc: et là le lien chaoss.)

Reply:
I've tried something but we could switch to temp.org if it's not enough ;)